engine/src/test/java/org/hibernate/validator/test/internal/engine/valuehandling/OptionalWithPrivateGetterTest.java

```diff
@@ -41,7 +42,7 @@ public void testConstraintCanBePlacedOnPrivateGetter() {
 	public void testConstraintOnPrivateGetterReturnsConstraintViolation() {
 		Validator validator = getValidator();
 
-		LocalDate[][] testData = { { LocalDate.MAX, LocalDate.MIN }, { LocalDate.now(), null }, { null, null } };
+		LocalDate[][] testData = { { LocalDate.MAX, LocalDate.MIN }, { LocalDate.now( ZoneId.systemDefault() ), null }, { null, null } };
```

Comment:
I do agree with you. I've looked at those bundled files and these Locale/TimeZone/Charset things comes from `jdk-unsafe`. As there seems to be nothing else (except locals and timezones,charsets) in those unsafe files I'd say it can be _safely_ :) removed.
Link to the signature files if needed: 
https://github.com/policeman-tools/forbidden-apis/tree/master/src/main/resources/de/thetaphi/forbiddenapis/signatures